src/agent.ts

```diff
@@ -378,6 +378,9 @@ export class Agent {
       // JWTs
       await interxn.processInteractionToken(token)
       await this.storage.store.interactionToken(token)
+    } else if (interxn.lastMessage.encode() === jwt) {
+      // @ts-ignore
+      await interxn._processToken(token)
```

Comment:
In the wallet we are relying fully on the event system for showing new interactions, in order to support deeplinks. This change is needed only to emit the "interactionUpdate" event if the same token is processed twice.

Reply:
`_processToken` is a private method, you should not access it directly

Reply:
also, `interaction._processToken(message, true)` will run twice: when you try to find interaction in a dictionary and here, the only difference is the parameter `fromStorage` being passed. 
What about emitting a different event, like `interactionResumed`, i.e.
```
if (!fromStorage) {
    const eventName = this.messages.length === 1 ? 'interactionCreated' : 'interactionUpdated'
    this.ctx.emit(eventName, this)
  } else {
    this.ctx.emit('interactionResumed', this)
  }